src/components/onboarding/steps.js

```diff
@@ -0,0 +1,225 @@
+      fontSize: '16px',
+      backgroundImage: 'linear-gradient(-224deg, #FF6236 14%, #C80039 100%)',
+      color: '#fff',
+      height: '60px',
+      width: '85%',
+      margin: '20px 0px 40px 0px',
+      borderRadius: '3px',
+      float: 'none',
+    },
+    footer: {
+      textAlign: 'center',
+      padding: '0px 90px 0px',
+    },
+    close: {
+      display: 'none',
+    },
+    button: {
+      display: 'none',
+    },
+  },
+  intro: {
+    fontSize: '15px',
+    borderRadius: '0',
+    color: '#000',
+    textAlign: 'center',
+    width: '486px',
+    arrow: {
+      display: 'none',
+    },
+    main: {
+      padding: '10px 90px 30px',
+    },
+    header: {
+      borderBottom: 'none',
+      fontFamily: 'gilroy,Open Sans,sans-serif',
+      fontSize: '28px',
+      padding: '80px 90px 20px',
+    },
+    skip: {
+      fontFamily: 'Open Sans,sans-serif',
+      color: '#74869B',
+      float: 'none',
+      margin: '0',
+    },
+    footer: {
+      textAlign: 'center',
+      padding: '0px 90px 0px',
+    },
+    close: {
+      display: 'none',
+    },
+    button: {
+      fontFamily: 'Open Sans,sans-serif',
+      backgroundImage: 'linear-gradient(224deg, #FF6236 14%, #C80039 100%)',
+      color: '#fff',
+      height: '60px',
+      width: '85%',
+      margin: '20px 0px 40px 0px',
+      fontSize: '16px',
+    },
+  },
+  outro: {
+    fontSize: '15px',
+    borderRadius: '0',
+    color: '#000',
+    textAlign: 'center',
+    width: '486px',
+    arrow: {
+      display: 'none',
+    },
+    main: {
+      padding: '10px 90px 30px',
+    },
+    header: {
+      borderBottom: 'none',
+      fontFamily: 'gilroy,Open Sans,sans-serif',
+      fontSize: '28px',
+      padding: '80px 90px 20px',
+    },
+    skip: {
+      display: 'none',
+    },
+    footer: {
+      textAlign: 'center',
+      padding: '0px 90px 0px',
+    },
+    close: {
+      display: 'none',
+    },
+    button: {
+      fontFamily: 'Open Sans,sans-serif',
+      backgroundImage: 'linear-gradient(224deg, #FF6236 14%, #C80039 100%)',
+      color: '#fff',
+      height: '60px',
+      width: '85%',
+      margin: '20px 0px 40px 0px',
+      fontSize: '16px',
+    },
+  },
+};
+
+export const steps = t => ([
+  {
+    title: t('Onboarding whenever you need'),
+    text: t('Go to "More" on your Sidebar to start the onboarding whenever you need.'),
+    selector: '#onboardingAnchor',
+    position: 'bottom',
+    style: styles.skip,
+  },
+  {
+    title: t('Welcome to Lisk Hub'),
+    text: t('Take a tour to see how it works. It should not take longer than 5 minutes'),
+    selector: '#onboardingAnchor',
+    position: 'bottom',
+    style: styles.intro,
+  },
+  {
+    title: t('Timeout'),
+    text: t('After 10 minutes of inactivity you will be logged out to prevent an unauthorized access to your Lisk ID. This will reset as soon as you become active again. You can also reset the timer anytime by clicking on the countdown.'),
```

Comment:
> After 10 minutes of not using your passphrase, your Lisk ID will be locked to prevent an unauthorized use of your Lisk ID. The timer resets when you send any transaction. You can also reset the timer by clicking "reset" when there are less than 5 minutes left.